src/common/runtime/wpt.ts

```diff
@@ -21,6 +21,7 @@ declare function done(): void;
 declare function assert_unreached(description: string): void;
 
 declare const loadWebGPUExpectations: Promise<unknown> | undefined;
+declare const shouldWebGPUCTSFailOnWarnings: Promise<boolean> | undefined;
```

Comment:
Does it need to be a promise? I think loadWebGPUExpectations is only a promise because it normally involves an import() call.

Reply:
Like, we should be able to just set it in cts.html (and in fact maybe this PR should add it to `src/common/templates/cts.html` as a reference, just like we have for loadWebGPUExpectations).

Reply:
I think a promise would be most flexible because we could also `import` it. That allows the browser test runner to simply switch a flag instead of generating a different .html file.

Chromium would have a single `import`:
```javascript
const loadWebGPUConfig = import('some/path/to/config.js');
const loadWebGPUExpectations = loadWebGPUConfig.then(c => c.expectations);
const shouldWebGPUCTSFailOnWarnings = loadWebGPUConfig.then(c => c.shouldWebGPUCTSFailOnWarnings);
```